extensions/amp-analytics/amp-analytics.md

```diff
@@ -328,12 +328,13 @@ The element visibility trigger can be configured for any AMP element or a docume
 
 Notice that selector can be used to only specify a single element, not a collection. The element can be either an [AMP extended  element](https://github.xom/ampproject/amphtml/blob/master/spec/amp-tag-addendum.md#amp-specific-tags) or a document root.
 
-The element visibility trigger waits for element's [`ini-load`](#initial-load-trigger) signal before matching the `visibilitySpec`.
+The element visibility trigger waits for element's [`ini-load`](#initial-load-trigger) signal before tracking element visibility. The signal that it waits can ben customized using the `waitFor` property in `visibilitySpec`.
```

Comment:
Maybe "The element visibility trigger waits for .... signal specified by the `waitFor` property in visibilitySpec. If not specified it defaults to `ini-load`. See `waitFor` docs for more details"? Or something like that...